k8s/backend-deployment.yaml

```
@@ -0,0 +1,130 @@
+        - |
+          # Set proper npm cache directory
+          export npm_config_cache=/tmp/.npm
+          export HOME=/tmp
+          
+          # Create app directory 
+          mkdir -p /tmp/app/src
+          
+          # List ConfigMap contents
+          echo "ConfigMap contents:"
+          find /source -type f -name "*.json" -o -name "*.ts" -o -name "*.js" | head -20
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)